packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -203,8 +207,8 @@ export class InputTimePicker
 
   @Watch("disabled")
   @Watch("readOnly")
-  handleDisabledAndReadOnlyChange(value: boolean): void {
-    if (!value) {
+  handleDisabledAndReadOnlyChange(open: boolean): void {
```

Comment:
This rename is misleading as the parameter will be whatever `disabled` or `readOnly` was set to (this watch handler is not watching `open`).